docs/astro/sql/operators/raw_sql.rst

```diff
@@ -0,0 +1,24 @@
+====================
```

Comment:
```suggestion
.. _run_raw_sql:

====================
```

Reply:
@pankajastro  Add the reference so it is easier for other sections to refer this operator.